lib/__snapshots__/index.test.js.snap

```diff
@@ -4,55 +4,80 @@ exports[`should work on real web data 1`] = `
 Array [
   "Google Analytics - https://www.google-analytics.com/urchin.js",
   "Google Fonts - https://fonts.googleapis.com/tr/css",
-  "Google Fonts - https://fonts.gstatic.com/stats/Ubuntu/normal/700",
-  "Other Google APIs/SDKs - https://www.google.com/xjs/_/js/md=1/k=xjs.qs.en_US.lV9-yIzYVQw.O/ck=xjs.qs.6H7if2Wkqlc.L.W.O/ct=zgms/am=ggAKAACARRj50xlGOaI4gH8JYMYBAAAm4DDB_gKoEE8ACIAAhK2EBA/rs=ACT90oHTsQ6_NdRrrQpQMC5y1T_UyuurBw",
-  "Google/Doubleclick Ads - https://stats.g.doubleclick.net/robots.txt",
+  "Google Fonts - https://fonts.gstatic.com/stats/Ubuntu/normal/700
```

Comment:
I'm not sure what happened here, but tests are failing because of all of these added newlines. diff should just be adding your great new facade :)

Reply:
I restored the file to it's original state since I don't have new urls

Reply:
So sorry for this mistake 😅